### DISCUSSION TITLE
Over write the same file

### DISCUSSION THREAD

**Discussion by NextGenerationHackers** [j4k0xb/webcrack#115] [General]
![image](https://github.com/user-attachments/assets/ee6e7d16-1868-4bb3-a46f-2156dcf8b1b5)

How Can I over write the same file but with the changes instead; I dont want the content to be outputted to another file as I have currently but I just want to over write the code that was decompiled.

**Comment by j4k0xb** [comment]
that's a limitation of the shell
when using `webcrack foo.js > foo.js` it immediately overwrites the file, no chance to read the content before

you can however add `&& move Decoded.js WarCommander.js` to achieve the same thing

**Reply by j4k0xb** [reply]
oh and `NODE_OPTIONS=` is only for Linux/Mac, I just updated the docs to make it work for Windows:
```cmd
set NODE_OPTIONS=--max-old-space-size=8192
webcrack bundle.js
```
